docs/spec/v1/gitrepositories.md

```diff
@@ -173,7 +174,7 @@ metadata:
   namespace: default
 type: Opaque
```

Comment:
This could be `kubernetes.io/tls` rather than `Opaque`.
```suggestion
type: kubernetes.io/tls
```

Reply:
This particular can't be a Secret of type TLS, since it could contain other info like username, ssh keys, etc.

Reply:
Ah, then:
```suggestion
type: Opaque # not kubernetes.io/tls
             # (because it can contain other info besides keys and certificates)
```
?

Reply:
don't think we need to explicitly call it out, since the default type is Opaque anyway and the reconciler does not check the type of the Secret (and we can't start being strict about the type now, since this API is v1, so no breaking changes)